lib/msf/core/db_manager/cred.rb

```diff
@@ -16,7 +16,7 @@ def creds(opts)
       query = query.includes(logins: [ :service, { service: :host } ])
 
       if opts[:type].present?
-        query = query.where('"metasploit_credential_privates"."type" = ?', opts[:type])
+        query = query.where('"metasploit_credential_privates"."type" = ?', opts[:type].to_s)
```

Comment:
Fixing a type coercion, where `opts[:type]` was a real Ruby object when it was expecting a string or primitive value - which broke Rails. Forcing `.to_s` ensures the string value is used